### ISSUE TITLE
CANape 15

### ISSUE THREAD

**Issue by tmheath** [zariiii9003/pycanape#5]
I'm trying to use the CANape api to talk to CANape version 15. I've noticed that you do not support 15, is this because it's just not a target or because something is different about the dll? I've tried windll and WinDLL (as used in this project). Initially I was going to use this library but found that CANape 15 was unsupported.

Thanks for any information

**Comment by zariiii9003** [comment]
I cannot test with CANape 15, but it might work. I'd say just try and report back.

**Comment by tmheath** [comment]
Thanks, I'll test it on current python with CANape15 monday, I don't have the software at home.
I'm currently writing a python lib in rust for this but if I'd just be replicating work then I'll scrap it in a heartbeat.
We use a handful of different CANape versions at work with more to come, and other tools. I was
going to make a universal multipurpose library, been having trouble with python and the dll but rust is working (I don't like the borrow checker, I like that the package manager and the build system are built in and tied together so there aren't issues handling libraries).

**Comment by tmheath** [comment]
@zariiii9003 It appears that canape 15 is at least trying to work with errors about functions not being found.
![image](https://github.com/zariiii9003/pycanape/assets/85769253/867118e5-7403-4769-a9d8-9febe9bda2ef)
Running more tests, will confirm

**Comment by tmheath** [comment]
Yeah, I think this is working but I'm going to have a couple pull requests from going over the initial documentation, thinking about adding __enter__ and __exit__ so that canape can just be called as a context manager so you don't have to worry about whether exit is called. I'll also submit a pr with canape15 to add to the documentation if this ends up fully working

**Comment by tmheath** [comment]
Also if you don't mind, I'd like to add a pdf target for the documentation so I can just download that instead of having use a web browser.

**Comment by zariiii9003** [comment]
Sounds good 👍

**Comment by meganmlee** [comment]
@tmheath @zariiii9003 How do you resolve the issue with the functions not being found? I've been trying with CANape 17 and 18 and not sure how to resolve.
